myhpom/static/myhpom/include/_upload_widget.scss

```diff
@@ -57,15 +57,16 @@
     @extend .h6;
 
     border: 1px solid black;
-    color: lightgray;
+    color: darkgray;
     display: inline-block;
     margin: 0.5rem 0;
     padding: 2rem;
     text-align: center;
     width: 100%;
 
     &:any-link {
-        color: lightgray;
+        color: darkgray;
+        cursor: pointer;
```

Comment:
I think this needs to go up into its parent selector, not just `:any-link`, since after `/dashboard/` itself none of the relevant items are links, just `button`s, and we need some kind of hover indication that they do something.

Reply:
ah, right right. Fixed.